### ISSUE TITLE
M-audio M-track solo Issues

### ISSUE THREAD

**Issue by souKani** [mdias/rs_asio#256]
I'm currently having issues with my new audio interface, not only on Rocksmith 2014, 3 at all; 
First of them, I can't change the sample rate:
![image](https://user-images.githubusercontent.com/98617499/151616011-ba4f08fc-4234-4a03-8f24-9ac55662d9e4.png)
This is the driver control panel, very poor in information; in line properties I just have 3 options, and they are:
- Channel 2 - 16 bits 44100 Hz (CD Quality)
- Channel 2 - 24 bits 44100 Hz (Studio Quality)
- Channel 2 - 32 bits 44100 Hz (Studio Quality)
And YES, it does have support for 48 kHz.
![image](https://user-images.githubusercontent.com/98617499/151617573-e02f2447-8e3e-4135-a223-05534817865c.png)

Here's the second one, I prefer to use my output(Headphone P2 connection) directly in the analog PC input. But becaus of the ASIO Output driver, I tried to connect it to the AU phones output and the use de Output channel 2.
![image](https://user-images.githubusercontent.com/98617499/151618417-8441edcd-6ca4-447a-815e-b298bc30069a.png)
![image](https://user-images.githubusercontent.com/98617499/151619246-9cf6b208-9e24-484f-954c-f77be081fdfe.png)

And then, it appears in the game:
![image](https://user-images.githubusercontent.com/98617499/151618255-1ea01e1b-dbd2-4c26-8360-33b59f8e2bd1.png)

The third issue I'm having, I think it's connected with the first, because I can't connect the guitar(instrument input) with the game, even with the drivers described in the logs in the correct input slot.
![image](https://user-images.githubusercontent.com/98617499/151619132-c6c1d443-252d-479e-aab8-b218a443f80e.png)
![image](https://user-images.githubusercontent.com/98617499/151619461-49846a46-062f-43c7-a8db-686d6a4c650e.png)

I want to play it so bad. I hope you guys can help me with that!

**Comment by mdias** [comment]
To solve your number 1 and 3 issues you'll have to get the interface to switch to 48kHz which you can't for some odd reason. Maybe try updating your interface drivers.

For using your computer's normal audio output instead of the interface, use:
```
[Config]
EnableWasapiOutputs=1
....

[Asio.Output]
Driver=
```

This will disable the ASIO output, and enable the WASAPI outputs which should allow the game to use the same output as it did without RS_ASIO.

**Comment by souKani** [comment]
> 

Thx! Now I can hear if the guitar is connected or not!
I tested the interface in another PC with Windows 7, and used the Windows 10 driver app, I was able to use the 48 kHz thing;
My PC is running 2 years without a format thing. I think there are so much drivers, conflict and smth like that and makes sense it stucks the sample rate. Maybe an older version of the driver works. Tomorrow I'll test it. Thanks for all!
I'll let the Issue open, after the tests are made I come back with more information.
